Doc/library/platform.rst

```diff
@@ -17,6 +17,13 @@
    section.
 
 
+.. note::
+
+   Calling a platform-specific method on a mismatched platform will return its 
+   default value. For example, calling :func:`win32_ver` on Linux will return
+   ``('', '', '', '')``.
```

Comment:
There are no methods, only functions.

I prefer to not document win32_ver() function result here, I would make the doc harder to maintain and I don't think that this example is helpful.

I'm not sure about "its default value": many functions return a tuple made of multiple values. Maybe say "default values" instead?

Reply:
Hmm good points, I was thinking a tuple to be one value but multiple makes more sense. To avoid specifying an example, would a vaguer one like `For example, on a mismatched platform, a function may return a tuple with empty string values` work, or better to just scrap it and leave it as `Calling a platform-specific method on a mismatched platform will return their default values.`?

Reply:
@vstinner ^ Any thoughts on which option to take?

Reply:
I prefer the first form. "default value" doesn't seem meaningful to me.